packages/orama/src/types.ts

```diff
@@ -18,9 +18,9 @@ export interface Schema extends Record<string, SearchableType | Schema> {}
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface Document extends Record<string, SearchableValue | Document | unknown> {}
 
-export type SearchableType = 'string' | 'number' | 'boolean'
+export type SearchableType = 'string' | 'number' | 'boolean' | 'string[]' | 'number[]' | 'boolean[]'
 
-export type SearchableValue = string | number | boolean
+export type SearchableValue = string | number | boolean | Array<string> | Array<number> | Array<boolean>
```

Comment:
Possibly a stupid question, why are we using `Array<T>` here and `T[]` in `SearchableType`?

Reply:
Agree. I think the linter should have catched this.